app/assets/javascripts/modules/SearchFilter.js

```diff
@@ -0,0 +1,31 @@
+define(['jquery'], function($) {
+
+  'use strict';
+
+  var $element = $('[data-search-filter]'),
+      $trigger = $('[data-search-filter-trigger]'),
+      $triggerIcon = $('[data-search-filter-icon]'),
+      $triggerHeading = $('[data-search-heading-second]'),
+      $target = $('[data-search-filter-target]');
+
+  $element.each(function() {
+    var $this = $(this);
+
+    $this.find($trigger).on('click', function(event) {
+      event.preventDefault();
+
+      // hide all of the filter sections on the page
+      $target.addClass('is-hidden');
+
+      // force the icons to revert to pointing left/closed
+      $triggerIcon.removeClass('search__triangle-icon--down');
+
+      // find the specific filter section and toggle its visibility and set the icon to open/down
+      $this.find($target).toggleClass('is-hidden');
```

Comment:
We should cache these as variables further up, so:

```js
var $elementTarget = $(this).find($target);
```

then...

```js
$elementTarget.toggleClass('is-hidden');
```

Although jQuery does do some caching internally, it just ensures that it doesn't need to search the DOM every time this is clicked